content/en/docs/tasks/administer-cluster/migrating-from-dockershim/find-if-you-affected.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Finding if you are affected by dockershim deprecation
+content_type: task 
+reviewers:
+- SergeyKanzhelev
+weight: 20
+---
+
+<!-- overview -->
+
+This is very unlikely that your application has a dependency on dockershim. In
+order to understand how your cluster or apps may acquire dependency on Docker as
+a container runtime, you need to understand what role dockershim plays when
+container is being scheduled on the node.
+
+## Container runtimes
+
+[Container runtime](/docs/setup/production-environment/container-runtimes/) is a software that can schedule Pods. Kubernetes is responsible for orchestration of application Pods, and uses CRI (container runtime interface) to communicate to container runtime to actually schedule these Pods and get their status.
```

Comment:
```suggestion
A [container runtime](/docs/concepts/containers/#container-runtimes) is software that can
execute the containers that make up a Kubernetes pod. Kubernetes is responsible for orchestration
and scheduling of Pods; on each node, the {{< glossary_tooltip text="kubelet" term_id="kubelet" >}}
uses the container runtime interface as an abstraction so that you can use any compatible
container runtime.
```